agent/src/main/scala/za/co/absa/atum/agent/model/MeasuresBuilder.scala

```diff
@@ -20,6 +20,9 @@ import za.co.absa.atum.agent.exception.AtumAgentException.MeasureException
 import za.co.absa.atum.agent.model.Measure._
 import za.co.absa.atum.model.dto
 
+/**
+ * This object provides a functionality to convert a DTO representation of measures to their internal representation.
```

Comment:
```suggestion
 * This object provides a functionality to convert a DTO representation of measures to the Agent's internal representation of this object.
```

or something similar?

Reply:
Don't see much of a difference. Let me know if you feel strongly about this, otherwise I suggest to keep it as it is.

Reply:
I think my suggestion makes it easier to read for someone not familiar with the codebase & it's more exact

Reply:
Ok. Changed.